pagopa/io-app-design-system
# chore: add new batch of pictograms

## Short description


<img src="https://github.com/pagopa/io-app-design-system/assets/6160324/d68634b8-3140-44f5-98a0-693d1052119b" width="100" />
<img src="https://github.com/pagopa/io-app-design-system/assets/6160324/ff25e4fc-1bf4-498c-95bb-1973f1e8235f" width="100" />
<img src="https://github.com/pagopa/io-app-design-system/assets/6160324/5a315223-3d7b-47da-a3ab-230fc1751f99" width="100" />

## List of changes proposed in this pull request
- New `eventClose` pictogram
- New `comunicationProblem` pictogram
- New `hello` pictogram
- Added alphabetical order for pictograms in the example app

## How to test
In the example app, navigate to the **Pictograms** page and check if the new pictograms are correctly visible.
